ratatui-core/src/text/line.rs

```diff
@@ -188,6 +190,9 @@ pub struct Line<'a> {
 
     /// The spans that make up this line of text.
     pub spans: Vec<Span<'a>>,
+
+    /// The overflow handling for this line of text.
+    pub overflow: Overflow<'a>,
```

Comment:
One big issue that this PR immediately brings to to the fore, is that we need to lock down the public fields of types in -core, otherwise this sort of change is a breaking change.